### ISSUE TITLE
How to customize transitions?

### ISSUE THREAD

**Issue by yang** [davidkpiano/flipping#10]
Thanks for the wonderful library!  How does one tweak the animation in play, such as easing/spring parameters, duration, etc.?

**Comment by davidkpiano** [comment]
You can pass parameters into:

```js
const flipping = new Flipping({
  duration: 300,
  easing: 'ease-in-out'
});
```

etc.

**Comment by yang** [comment]
Is it possible to customize this on a per-element (flip-key) level?

**Comment by yang** [comment]
Or rather, per (element, property)

**Comment by davidkpiano** [comment]
No, but that's a good idea. I'm still thinking of the proper API for that, it will likely be state-machine based.

**Comment by phelma** [comment]
<https://github.com/davidkpiano/flipping/pull/14>
